packages/launcher/lib/browsers.ts

```diff
@@ -1,14 +1,161 @@
+// to determine if the browser is supported by Cypress.
+export const validateMinVersion = (browser: FoundBrowser): BrowserValidatorResult => {
+  const minSupportedVersion = browser.minSupportedVersion
+  const majorVersion = browser.majorVersion
+
+  if (majorVersion && minSupportedVersion && parseInt(majorVersion) < minSupportedVersion) {
+    return {
+      isSupported: false,
+      warningMessage: `Cypress does not support running ${browser.displayName} version ${majorVersion}. To use ${browser.displayName} with Cypress, install a version of ${browser.displayName} newer than or equal to ${minSupportedVersion}.`,
+    }
+  }
+
+  return {
+    isSupported: true,
+  }
+}
 
 /** list of the browsers we can detect and use by default */
 
+export const browsers: Browser[] = [
+  {
+    name: 'chrome',
+    family: 'chromium',
+    channel: 'stable',
+    displayName: 'Chrome',
+    versionRegex: /Google Chrome (\S+)/m,
+    binary: ['google-chrome', 'chrome', 'google-chrome-stable'],
+    minSupportedVersion: MIN_CHROME_VERSION,
+  },
+  {
+    name: 'chromium',
+    family: 'chromium',
+    // technically Chromium is always in development
+    channel: 'stable',
+    displayName: 'Chromium',
+    versionRegex: /Chromium (\S+)/m,
+    binary: ['chromium-browser', 'chromium'],
+    minSupportedVersion: MIN_CHROME_VERSION,
+  },
+  {
+    name: 'chrome',
+    family: 'chromium',
+    channel: 'beta',
+    displayName: 'Chrome Beta',
+    versionRegex: /Google Chrome (\S+) beta/m,
+    binary: 'google-chrome-beta',
+    minSupportedVersion: MIN_CHROME_VERSION,
+  },
+  {
+    name: 'chrome',
+    family: 'chromium',
+    channel: 'canary',
+    displayName: 'Canary',
+    versionRegex: /Google Chrome Canary (\S+)/m,
+    binary: 'google-chrome-canary',
+    minSupportedVersion: MIN_CHROME_VERSION,
+  },
+  {
+    name: 'firefox',
+    family: 'firefox',
+    channel: 'stable',
+    displayName: 'Firefox',
+    // Mozilla Firefox 70.0.1
+    versionRegex: /^Mozilla Firefox ([^\sab]+)$/m,
+    binary: 'firefox',
+    minSupportedVersion: MIN_FIREFOX_VERSION,
+    validator: (browser: FoundBrowser, platform: NodeJS.Platform): BrowserValidatorResult => {
+      // Firefox 101 and 102 on Windows features a bug that results in Cypress being unable
+      // to connect to the launched browser. A fix was first released in stable 103.
+      // See https://github.com/cypress-io/cypress/issues/22086 for related info.
+
+      if (platform === 'win32' && browser.majorVersion && ['101', '102'].includes(browser.majorVersion)) {
+        return {
+          isSupported: false,
+          warningMessage: `Cypress does not support running ${browser.displayName} version ${browser.majorVersion} on Windows due to an unpatched defect. To use ${browser.displayName} with Cypress on Windows, install version 103 or newer.`,
+        }
+      }
+
+      return validateMinVersion(browser)
+    },
```

Comment:
I did not utilize the same validator for the dev/nightly channels as the impacts are much more undefined; some 100/101/102/103/104 versions will be impacted, some won't.

Reply:
In theory we could use the `info` property to set more a of "ymmv" version of the message for those channels, but it is not actually rendering in the UI at the moment: https://github.com/cypress-io/cypress/issues/21769

This is early in my list to ask to prioritize after the search work is done.

Reply:
Thanks for the heads up Mark, using `info` (when it works) would be perfect